docs/sources/administration/plugin-management/index.md

```diff
@@ -103,53 +103,43 @@ To browse for available plugins:
 
 ### Install a plugin
 
-To install a plugin:
+To install a plugin you have multiple possibilities, the regular way is using Grafana UI.
```

Comment:
The most common way to install a plugin is through the Grafana UI, but alternative methods are also available.

Reply:
thanks, adjusted